packages/nx/src/command-line/generate.ts

```diff
@@ -372,6 +372,8 @@ export async function generate(cwd: string, args: { [k: string]: any }) {
       }
 
       const task = await implementation(host, combinedOpts);
+      host.seal();
```

Comment:
There are more places where a `Tree` is used.. we should lock the tree there too. Search for `flushChanges`.